.github/workflows/check-templates.yml

```diff
@@ -52,6 +58,21 @@ jobs:
           ref: ${{ steps.branch-name.outputs.head_ref_branch }}
           fetch-depth: 0
 
+      - name: Copy repo into workspace
+        run: |
```

Comment:
use checkout action instead and use inputs to set up specific location : 
`path`: input argument